gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/GremlinProcessRunner.java

```diff
@@ -54,7 +56,7 @@ public void runChild(final FrameworkMethod method, final RunNotifier notifier) {
             } catch (Throwable e) {
                 if (validateForGraphComputer(e)) {
                     eachNotifier.fireTestIgnored();
-                    System.err.println(e.getMessage());
+                    logger.error(e.getMessage());
                     ignored = true;
```

Comment:
@dkuppitz perhaps on merge we could change this to WARN level?  this situation doesn't really indicate a "failure" does it?